samples/word/20-lists/manage-formatting.yaml

```diff
@@ -0,0 +1,132 @@
+order: 2
+id: word-lists-manage-formatting
+name: List formatting
+description: Shows how to format a list.
+author: OfficeDev
+host: WORD
+api_set:
+    WordApi: '1.3'
+script:
+    content: |
+        $("#insert-list").click(() => tryCatch(insertFormatList));
+        $("#setup").click(() => tryCatch(setup));
+
+        async function insertFormatList() {
+          // Insert a list starting with the third paragraph then set numbering and bullet types of the list items.
+          await Word.run(async (context) => {
+            const paragraphs = context.document.body.paragraphs;
+            paragraphs.load("$none"); // No properties needed.
+
+            await context.sync();
+
+            // Start new list using the third paragraph.
+            const list = paragraphs.items[2].startNewList();
+            list.load("$none"); // No properties needed.
+
+            await context.sync();
+
+            // To add new items to the list, use Start or End on the insertLocation parameter.
+            list.insertParagraph("New list item at the start of the list", "Start");
+            const paragraph = list.insertParagraph("New list item at the end of the list (5th level)", "End");
```

Comment:
I find this string confusing "New list item at the end of the list (5th level)", since it sounds like the end of the list is the fifth level.